algorithms/polish.sbl

```diff
@@ -0,0 +1,229 @@
+stringescapes {}
+
+stringdef a' '{U+0105}' 
+stringdef e' '{U+0119}' 
+stringdef l' '{U+0142}' 
+stringdef s' '{U+015b}' 
+stringdef c' '{U+0107}' 
+stringdef o' '{U+00f3}' 
+
+
+
+
+externals (stem)
+integers (pos)
+
+
+routines (remove_nouns remove_dimunitive remove_adjective_ends remove_verbs_ends remove_adverbs_ends 
+	remove_plural_forms remove_general_ends)
+
+
+define remove_nouns as (
+			($(len > 7)
+			test ($pos = (len - 5)
```

Comment:
The 5 here assumes that each character counts as `1` which isn't true when we're working in UTF-8 - it's assumptions such as this which are causing the tests for fail when we're working in UTF-8.

I think this is going to need to significant restructuring to fix satisfactorily.